src/test/java/redis/clients/jedis/UnifiedJedisTest.java

```diff
@@ -0,0 +1,263 @@
+package redis.clients.jedis;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.sameInstance;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import redis.clients.jedis.commands.ProtocolCommand;
+import redis.clients.jedis.mocked.unified.UnifiedJedisTestBase;
+
+/**
+ * These tests are part of the mocked tests for {@link UnifiedJedis}, but, due to {@code protected}
+ * visibility of some methods, they must reside in the same package as the tested class.
```

Comment:
Could you mention which ones? We can check if making those `public` makes sense.

Reply:
Actually I was wrong, it's not `UnifiedJedis`. It is about:
* `CommandObjects.commandArguments(ProtocolComand)`
* `CommandArguments.processKey(*)`